src/main/java/bio/terra/resourcemanagement/metadata/google/GoogleProjectRequest.java

```diff
@@ -34,4 +36,15 @@ public GoogleProjectRequest serviceIds(List<String> serviceIds) {
         this.serviceIds = serviceIds;
         return this;
     }
+
+    // map of identities to list of iam permissions (a list of things the id'ed user can do)
```

Comment:
It's the other way around, `role` -> `identities`. We should also rename this to something like `roleIdentityMapping`.

Reply:
I'm unclear on the domain of the data here. Is this simply holding SAM state of which users are in the stewards group? What is in the map when we add the creating custodian (or all users with custodian access)? Seems like it shouldn't be driven by role, but I may be missing the whole point...